ies-route53/recipes/add.rb

```diff
@@ -3,11 +3,13 @@
 node.force_override['route53']['fog_version'] = '1.33.0'  # ~FC019
 include_recipe 'route53'
 
+instance = get_instance
+
 host_name = get_hostname(node, true)
 stack_name = get_normalized_cluster_name(node)
 zone_name = node['ies-route53']['zone']['name']
-region_id = node['opsworks']['instance']['region']
-public_ip = node['opsworks']['instance']['ip']
+region_id = instance['region']
```

Comment:
@till, The EasyBib.easybib.get_instance function will return `nil` not run inside Opsworks. We can not rely on it always having a key named `region`, either.